subsurface-helper.cpp

```diff
@@ -61,8 +61,20 @@ void exit_ui()
 
 void run_ui()
 {
-
 #ifdef SUBSURFACE_MOBILE
+#if defined(Q_OS_ANDROID)
+	if (getAndroidHWInfo().contains("/OnePlus/")) {
+		QFontDatabase db;
+		int id = QFontDatabase::addApplicationFont(":/fonts/Roboto-Regular.ttf");
+		QString family = QFontDatabase::applicationFontFamilies(id).at(0);
+		QFont newDefaultFont;
+		newDefaultFont.setFamily(family);
+		(static_cast<QApplication *>(QCoreApplication::instance()))->setFont(newDefaultFont);
```

Comment:
That cast is ugly. If I read the docs correctly, you can either user `QApplication::instance()` or `qApp` without cast.

Reply:
stupidly we have a qApp macro that conflicts with the "normal" way of doing this :-(
Yes, the right solution would be to change that macro. I'll add fixing that to my long list and merge as is (as you suggested).

Reply:
> stupidly we have a qApp macro that conflicts with the "normal" way of doing this :-(

`git grep qApp` just shows a number of normal uses of `qApp`.

Reply:
is there a macro?

Reply:
> is there a macro?

random question on an eight months old PR?